core/app/modules/kit-library/assets/js/hooks/use-kits.js

```diff
@@ -149,6 +150,13 @@ export default function useKits( initialQueryParams = {} ) {
 		[ query.data, queryParams ]
 	);
 
+	const selectedTaxonomies = useSelectedTaxonomies( queryParams.taxonomies );
+
+	const isFilterActive = useMemo(
+		() => queryParams.search || !! selectedTaxonomies.length,
+		[ queryParams ]
+	);
+
```

Comment:
Maybe put it in the `useSelectedTaxonomies` hook and return an object? 
 ```JS
const { selectedTaxonomies , isFilterActive } = useSelectedTaxonomies( queryParams );
````

P.S. I don't really know if they are related, so maybe you shouldn't...

Reply:
Not related in this case, the filter can be active if the search is active or if the taxonomies filter is active.